src/core/agent/agent.ts

```diff
@@ -59,11 +59,11 @@ const agentModules = {
     ? { sqliteStorage: new SqliteStorageModule() }
     : { ionicStorage: new IonicStorageModule() }),
   signify: new SignifyModule(),
-  mediationRecipient: new MediationRecipientModule({
-    mediatorInvitationUrl:
-      "https://dev.mediator.cf-keripy.metadata.dev.cf-deployments.org/invitation?oob=eyJAdHlwZSI6Imh0dHBzOi8vZGlkY29tbS5vcmcvb3V0LW9mLWJhbmQvMS4xL2ludml0YXRpb24iLCJAaWQiOiI0YmU0NDk1OC01YmJhLTRlYTMtYjY2Zi05NWFlNDQ3ZjY0NjUiLCJsYWJlbCI6IkFyaWVzIEZyYW1ld29yayBKYXZhU2NyaXB0IE1lZGlhdG9yIiwiYWNjZXB0IjpbImRpZGNvbW0vYWlwMSIsImRpZGNvbW0vYWlwMjtlbnY9cmZjMTkiXSwiaGFuZHNoYWtlX3Byb3RvY29scyI6WyJodHRwczovL2RpZGNvbW0ub3JnL2RpZGV4Y2hhbmdlLzEuMCIsImh0dHBzOi8vZGlkY29tbS5vcmcvY29ubmVjdGlvbnMvMS4wIl0sInNlcnZpY2VzIjpbeyJpZCI6IiNpbmxpbmUtMCIsInNlcnZpY2VFbmRwb2ludCI6Imh0dHBzOi8vZGV2Lm1lZGlhdG9yLmNmLWtlcmlweS5tZXRhZGF0YS5kZXYuY2YtZGVwbG95bWVudHMub3JnIiwidHlwZSI6ImRpZC1jb21tdW5pY2F0aW9uIiwicmVjaXBpZW50S2V5cyI6WyJkaWQ6a2V5Ono2TWtzQ3Y3TUZLa1UyeVlyeUdzVEd2MWl3U01GcmRSMVd2V3dkbjQ3NFk3emgxVSJdLCJyb3V0aW5nS2V5cyI6W119LHsiaWQiOiIjaW5saW5lLTEiLCJzZXJ2aWNlRW5kcG9pbnQiOiJ3c3M6Ly9kZXYubWVkaWF0b3IuY2Yta2VyaXB5Lm1ldGFkYXRhLmRldi5jZi1kZXBsb3ltZW50cy5vcmciLCJ0eXBlIjoiZGlkLWNvbW11bmljYXRpb24iLCJyZWNpcGllbnRLZXlzIjpbImRpZDprZXk6ejZNa3NDdjdNRktrVTJ5WXJ5R3NUR3YxaXdTTUZyZFIxV3ZXd2RuNDc0WTd6aDFVIl0sInJvdXRpbmdLZXlzIjpbXX1dfQ",
-    mediatorPickupStrategy: MediatorPickupStrategy.Implicit,
-  }),
+  // mediationRecipient: new MediationRecipientModule({
```

Comment:
Once done we should merge develop into this to pick up these changes and multi-sig etc